.gitignore

```diff
@@ -35,4 +35,8 @@ yarn-error.log*
 *.tsbuildinfo
 .env*
 
-*.log
\ No newline at end of file
+*.log
+package-lock.json
+package.json
+functions/yarn.lock
+yarn.lock
```

Comment:
we should be committing and not ignoring the yarn.lock files

Reply:
i have no clue how that file was created, I just wanted create a image folder with an image

Reply:
thats fine, lets revert the changes to this file

Reply:
okay